mcsniperpy/sniper.py

```diff
@@ -155,10 +155,15 @@ async def run(self, target=None, offset=None):
                 "init_path"), "accounts.txt")
         )
 
-        droptime = await {"kqzz_api": api_timing, "namemc": namemc_timing}.get(
+        # droptime = await {"kqzz_api": api_timing, "namemc": namemc_timing}.get(
+        #     self.timing_system, "kqzz_api"
+        # )(target, self.session)
+        
+        droptime = await {"kqzz_api": api_timing, "namemc": teun_timing}.get(
```

Comment:
Suggest making a new key for teun api and keep namemc in case it does come back up, or teun goes down and namemc is up.

Reply:
Also make sure you test it and it works, check response and see if it is hitting teuns api

Reply:
Agreed on both of those things, Gobbo.

Reply:
NameMC timing should definitely still exist

Reply:
I did test the `teun_timing` function - it works. I don't quite know how the config system works, but I guess you can change where it gets the timing from? I agree - nameMC timing should still exist.